infrastructure/game-client-startup.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Script to install applications for the Game Client VM 
+
+export DEBIAN_FRONTEND=noninteractive
+
+wget -O "/usr/share/keyrings/xpra.asc" https://xpra.org/xpra.asc
+cd /etc/apt/sources.list.d 
+wget https://raw.githubusercontent.com/Xpra-org/xpra/master/packaging/repos/bookworm/xpra.sources
+sudo apt update -y
+cd -
+
+curl -OL https://github.com/LizardByte/Sunshine/releases/download/v0.21.0/sunshine-debian-bookworm-amd64.deb
+sudo apt install -f ./sunshine-debian-bookworm-amd64.deb -y
+sudo apt install xfce4 xfce4-goodies tightvncserver dbus-x11 gcc make linux-headers-$(uname -r) software-properties-common mesa-utils golang libgl1-mesa-dev xorg-dev xpra -y 
+
+wget https://developer.download.nvidia.com/compute/cuda/12.3.1/local_installers/cuda_12.3.1_545.23.08_linux.run
+sudo sh cuda_12.3.1_545.23.08_linux.run --silent
+
+sudo apt update -y
+
+#########
+#
+# Startup script to install Chrome remote desktop and a desktop environment.
+#
+# See environmental variables at then end of the script for configuration
+#
+
+function install_desktop_env {
+  PACKAGES="desktop-base xscreensaver dbus-x11"
+
+  if [[ "$INSTALL_XFCE" != "yes" && "$INSTALL_CINNAMON" != "yes" ]] ; then
+    # neither XFCE nor cinnamon specified; install both
+    INSTALL_XFCE=yes
+    INSTALL_CINNAMON=yes
+  fi
+
+  if [[ "$INSTALL_XFCE" = "yes" ]] ; then
+    PACKAGES="$PACKAGES xfce4"
+    echo "exec xfce4-session" > /etc/chrome-remote-desktop-session
+    [[ "$INSTALL_FULL_DESKTOP" = "yes" ]] && \
+      PACKAGES="$PACKAGES task-xfce-desktop"
+  fi
+
+  if [[ "$INSTALL_CINNAMON" = "yes" ]] ; then
```

Comment:
Curious why we need anything other than XFCE? Do we need a choice?

I think we can be opinionated here and give ourselves the narrowest path to support. WDYT?